src/pl/sys_package/ob_dbms_workload_repository.cpp

```diff
@@ -434,6 +436,24 @@ int ObDbmsWorkloadRepository::generate_ash_report_text(
       }
     }
 
+    if (OB_SUCC(ret)) {
+      if (ash_begin_time != ash_report_params.user_input_ash_begin_time) {
```

Comment:
`ash_begin_time` is queried from gv$active_session_history, which is very unlikely equals to `user_input_ash_begin_time`, even if gv$active_session_history covers user input time period

Reply:
we can remove this condition

Reply:
remove how?